packages/trader/src/AppV2/Components/RiskManagementInfoModal/__tests__/risk-management-info-modal.spec.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import RiskManagementInfoModal from '../risk-management-info-modal';
+
+jest.mock('@deriv/quill-icons', () => ({
+    LabelPairedCircleInfoSmRegularIcon: () => <svg />,
+}));
+
+describe('RiskManagementInfoModal', () => {
+    const headerContent = 'Risk Management Info';
+    const bodyContent = 'This is the body content of the modal.';
+    const infoMessage = 'Additional info message';
+
+    it('should render the button and modal content correctly', () => {
+        render(
+            <RiskManagementInfoModal
+                header_content={headerContent}
+                body_content={bodyContent}
+                info_message={infoMessage}
+            />
+        );
+
+        const button = screen.getByRole('button');
+        expect(button).toBeInTheDocument();
+
+        expect(screen.queryByText(headerContent)).not.toBeInTheDocument();
+    });
+
+    it('should toggle the modal visibility when the button is clicked', () => {
+        render(
+            <RiskManagementInfoModal
+                header_content={headerContent}
+                body_content={bodyContent}
+                info_message={infoMessage}
+            />
+        );
+
+        const button = screen.getByRole('button');
+        fireEvent.click(button);
```

Comment:
Can we use `userEvent`? We have an agreement to use it instead of `fireEvent` we it's possible